python/smashbox/utilities/__init__.py

```diff
@@ -668,20 +673,20 @@ def check_groups(num_groups=None):
             fatal_check(result, 'Group %s not found' % group_name)
 
 
-def expect_modified(fn, md5, comment=''):
+def expect_modified(fn, md5):
     """ Compares that the checksum of two files is different
     """
     actual_md5 = md5sum(fn)
     error_check(actual_md5 != md5,
-                "md5 of modified file %s did not change%s: expected %s" % (fn, comment, md5))
+                "md5 of modified file %s did not change: expected %s, got %s" % (fn, md5, actual_md5))
```

Comment:
Please pull the most recent version to your local repo and then create this pull request again. This change and the one below are not part of your patch but your local repo out of date.